.travis.yml

```diff
@@ -1,15 +1,14 @@
-os:
-- osx
+os: osx
+osx_image: xcode9.2
 script:
-- rm -rf /usr/local/Homebrew
-- travis_wait 60 rvm implode --force
-- ./script/boxen --no-fde --stealth --no-pull --debug --profile --login $BOXEN_GITHUB_LOGIN --token $BOXEN_GITHUB_TOKEN || test $? -eq 2
```

Comment:
We should probably leave `--stealth` here to prevent it opening issues on the repository.